package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "apostrophe",
-  "version": "2.100.1",
+  "version": "2.100.2",
```

Comment:
Wouldn't this be a minor version bump?

Reply:
The feature always existed in a busted form, so I regard it as a bug fix. Semantics but I'm good with it.